releasenotes/notes/star-algebra-mixin-ef201e21ab5d7fdd.yaml

```diff
@@ -0,0 +1,7 @@
+---
+upgrade:
+  - |
+    The internal API of the :class:`qiskit.opflow.OperatorBase` abstract class
+    has been changed to use :class:`~qiskit.opflow.mixins.StarAlgebraMixin` ported from
+    qiskit-nature.
```

Comment:
```suggestion
    has been changed to use :class:`~qiskit.opflow.mixins.StarAlgebraMixin` and
    :class:`~qiskit.opflow.mixins.TensorMixin` ported from qiskit-nature.
```

Reply:
Thanks. I forgot to mention the TensorMixin. TensorMixin is new added.